app/src/main/java/net/bible/service/db/DatabaseContainer.kt

```diff
@@ -1050,6 +1050,17 @@ private val MIGRATION_64_65_sync_group = object : Migration(64, 65) {
         }
     }
 }
+private val MIGRATION_65_66_add_Xrefs_option = object : Migration(65, 66) {
+    override fun doMigrate(db: SupportSQLiteDatabase) {
+        db.apply {
+            val colDefs = listOf<String>("`text_display_settings_showXrefs` INTEGER DEFAULT NULL")
+            colDefs.forEach {
+                execSQL("ALTER TABLE `Workspace` ADD COLUMN $it")
+                execSQL("ALTER TABLE `PageManager` ADD COLUMN $it")
```

Comment:
TODO: need to set value of xrefs option initially to be the same as footnote option value currently is.

Reply:
This is done now.